rascal2/ui/view.py

```diff
@@ -21,11 +22,18 @@ def __init__(self):
         self.undo_view.setWindowIcon(window_icon)
         self.undo_view.setAttribute(QtCore.Qt.WidgetAttribute.WA_QuitOnClose, False)
 
+        self.presenter = MainWindowPresenter(self)
+
+        # TODO Remove this when the startup windows are added! just here to 'bootstrap' controls creation for the widget
+        from RATapi import Controls  # noqa
+
+        self.presenter.model.controls = Controls()
+
         self.mdi = QtWidgets.QMdiArea()
         # TODO replace the widgets below
         self.plotting_widget = QtWidgets.QWidget()
         self.terminal_widget = QtWidgets.QWidget()
-        self.controls_widget = QtWidgets.QWidget()
+        self.controls_widget = ControlsWidget(self.presenter)
```

Comment:
For widgets, I recommend that we pass in the parent view instead i,e. `ControlsWidget(self)` then the presenter can be grabbed in the class, although not important just a a structural thing. As shown below,
```
class ControlsWidget(QtWidgets.QWidget):
    """Widget for editing the Controls object."""

    def __init__(self, parent):
        super().__init__()
        self.presenter = parent.presenter
```